Modules/cjkcodecs/multibytecodec.c

```diff
@@ -589,6 +589,12 @@ _multibytecodec_MultibyteCodec_encode_impl(MultibyteCodecObject *self,
     PyObject *errorcb, *r, *ucvt;
     Py_ssize_t datalen;
 
+    if(self->codec->encinit != NULL){
+        if(!Py_CHECKWRITE(self)){
+            return PyErr_WriteToImmutable(self);
+        }
+    }
```

Comment:
This file has new `Py_CHECKWRITE` but doesn't register the type as freezable. Is this declared somewhere else?